main.c

```diff
@@ -168,7 +168,10 @@ static void init()
    serial_module_init();
    write_log("OK\n");
 
-   sprintf(temp_buf, "Opening COM%i... ", comport.number + 1);
+   if (comport.number >= 1000)
```

Comment:
I think open_comport() should actually do this reporting, and should lose the "COM" stuff: it should say specifically what device path it's opening.

Reply:
I implemented the minimum set of code to enable the PTS support and differentiate the messages basing on the constraint that at the moment `comport.number` just stores an integer, disregarding the device type. Notice that USB does use the same strategy at the moment. I think that any additional refactoring to support the device type in the configuration should be part of a new release.

Reply:
Understood. I look forward to the improvements!

Reply:
Thanks!